src/ui/units/dash/containers/Dialogs/DialogRelations/DialogRelations.tsx

```diff
@@ -268,6 +276,18 @@ const DialogRelations = (props: DialogRelationsProps) => {
         [changedWidgets, preparedRelations, handleAliasesClick],
     );
 
+    /**
+     * Clear invalid aliases, which don't below to any dash tab field
+     */
+    const handleInvalidAliasesClear = React.useCallback(() => {
+        const filteredAliases = aliases[DEFAULT_ALIAS_NAMESPACE].map((aliasRow: string[]) => {
+            return aliasRow.filter((item) => !showInvalidAliases?.includes(item));
```

Comment:
Maybe to put the map's callback in a separate function for better readability?

Reply:
Map function is very short, don't think it is necessary here. There are lots of other functions much bigger than that one which are moved to helpers files (see other files in DialogRelations).